packages/cli/test/integration/credentials.api.test.ts

```diff
@@ -348,7 +349,8 @@ test('PATCH /credentials/:id should fail if cred not found', async () => {
 
 test('PATCH /credentials/:id should fail with missing encryption key', async () => {
 	const mock = jest.spyOn(UserSettings, 'getEncryptionKey');
-	mock.mockResolvedValue(undefined);
+	mock.mockRejectedValue(new Error(RESPONSE_ERROR_MESSAGES.NO_ENCRYPTION_KEY));
```

Comment:
should we also check for the correct messages in the tests?
atm we just check for the `500` status-code but not for what actually gets returned to the user as the message